app/src/home/res/values/strings.xml

```diff
@@ -36,5 +36,9 @@
     <string name="app_name">Let\'s Connect!</string>
     <string name="actionbar_title">Let\'s Connect!</string>
     <string name="add_other_address">Add Provider</string>
-
+    <!-- 
+         text hack to not display "Institute Access" because that has no 
```

Comment:
This is not a hack, but commonly done in Android (translations work like this too).
So feel free to remove this apology :)